client/src/Pages/IndexPage/index.scss

```diff
@@ -0,0 +1,128 @@
+$primary-color: #6b46c1;
+$secondary-color: #e53e3e;
+$background-color: #f3f4f6;
+$border-radius: 0.5rem;
+$gap: 1.5rem;
+
+@mixin button-styling {
+  border: 1px solid $primary-color;
+  background-color: white;
+  color: #4a5568;
+  padding: 0.25rem 0.5rem;
+  cursor: pointer;
+  border-radius: $border-radius;
+  transition: background-color 0.3s, color 0.3s;
+
+  &:hover {
+    background-color: $background-color;
+  }
+}
```

Comment:
same code as above

Reply:
Done with the required changes. Please check if anything left